package.json

```diff
@@ -13,16 +13,16 @@
   "scripts": {
     "audit:resolve": "SHELL=sh resolve-audit",
```

Comment:
would it be better to put ` --production` here to avoid having an enormous size of `audit-resolve.json` file?

Reply:
@lewisdaly what do you think?

Reply:
Yes please. That's what we're doing elsewhere. But feel free to do that in the next pr you make - no need to do it now if your audit check is passing